book/jsonrpc/intro.md

```diff
@@ -58,6 +58,16 @@ To enable JSON-RPC namespaces on the HTTP server, pass each namespace separated
 reth node --http --http.api eth,net,trace
 ```
 
+You can pass the `all`` flag to enable all JSON-RPC namespaces on the HTTP server:
+
```

Comment:
this should list that this is convenient for `admin,debug,eth,net,trace,txpool,web3,rpc`

Reply:
Good point, updated!